server/wizard/steps/step_goto_zoom_marketplace.go

```diff
@@ -0,0 +1,44 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow/steps"
+)
+
+const (
+	stepNameGotoZoomMarketplace = "goto_zoom_marketplace"
+
+	stepTitleGotoZoomMarketplace = "Let's create an app in Zoom!"
+
+	stepDescriptionGotoZoomMarketplace = `Next we're going to create a new app in your Zoom account.
+
+1. Go to [https://marketplace.zoom.us](https://marketplace.zoom.us) and log in using a Zoom admin account.
+2. In the top right corner of the screen, select **Develop** and then **Build App**.
```

Comment:
@aaronrothschild Any ideas on how I can get a self-hosted zoom server to test this out on?

Reply:
If I remember correctly, it is not exactly self hosted, but custom url. Like `companyName.zoom.us`. I think almost any company account is able to do this. I think (if you have the creds) you could use the mattermost one. If not, maybe aaron has anything else you can use.